mfc.sh

```
@@ -74,31 +50,31 @@ if [ "$1" '==' 'load' ] && [ "$2" != "--help" ] && [ "$2" != "-h" ]; then
     fi
     shift; . "$(pwd)/toolchain/bootstrap/modules.sh" $@; return
 elif [ "$1" '==' "lint" ] && [ "$2" != "--help" ] && [ "$2" != "-h" ]; then
-    . "$(pwd)/toolchain/bootstrap/python.sh"
+    . "$(pwd)/toolchain/bootstrap/python.sh" "$@"
 
     shift; . "$(pwd)/toolchain/bootstrap/lint.sh"    $@; exit 0
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)